clients/python/agentic-sandbox-client/sandbox_router/sandbox_router.yaml

```
@@ -0,0 +1,121 @@
+metadata:
+  name: external-http-gateway
+  namespace: default
+spec:
+  gatewayClassName: gke-l7-global-external-managed # Specific to GKE; creates all necessary GatewayClass resources
+  listeners:
+  - name: http
+    protocol: HTTP
+    port: 80
+    allowedRoutes:
+      namespaces:
+        from: Same
+---
+# A ClusterIP Service to provide a stable endpoint for the router pods.
+apiVersion: v1
+kind: Service
+metadata:
+  name: sandbox-router-svc
+  namespace: default
+spec:
+  type: ClusterIP
+  selector:
+    app: sandbox-router
+  ports:
+  - name: http
+    protocol: TCP
+    port: 8080 # The port the service will listen on
+    targetPort: 8080 # The port the router container listens on (from the sandbox_router/Dockerfile)
+---
+# The Deployment to manage and run the router pods.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sandbox-router-deployment
+  namespace: default
+spec:
+  replicas: 2 # Run at least two replicas for high availability
+  selector:
+    matchLabels:
+      app: sandbox-router
+  template:
+    metadata:
+      labels:
+        app: sandbox-router
+    spec:
+      # Ensure pods are spread across different zones for HA
+      topologySpreadConstraints:
+        - maxSkew: 1
+          topologyKey: topology.kubernetes.io/zone
+          whenUnsatisfiable: ScheduleAnyway
+          labelSelector:
+            matchLabels:
+              app: sandbox-router
+      containers:
+      - name: router
+        # IMPORTANT: Replace this with the full path to the router image you pushed.
+        # sed -i "s|IMAGE_PLACEHOLDER|${SANDBOX_ROUTER_IMG}|g" sandbox_router.yaml
+        image: IMAGE_PLACEHOLDER
+        ports:
+        - containerPort: 8080
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+          initialDelaySeconds: 10
+          periodSeconds: 10
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+          initialDelaySeconds: 5
+          periodSeconds: 5
+        resources:
+          requests:
+            cpu: "250m"
+            memory: "512Mi"
+          limits:
+            cpu: "1000m"
+            memory: "1Gi"
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+---
+# The HTTPRoute to connect the Gateway to the router service.
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: sandbox-router-route
+  namespace: default
+spec:
+  parentRefs:
+  - name: external-http-gateway # This must match the name of the Gateway
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /
+    backendRefs:
+    - name: sandbox-router-svc # Route all traffic to the router service
+      port: 8080
+---
+# HealthCheckPolicy tells the GKE Gateway to use the router app "/healthz" endpoint for its health
+# checks instead of the default "/".
+apiVersion: networking.gke.io/v1 # Specific to GKE; not yet available in gateway.networking.k8s.io
```

Will there any plan to make it be available in gateway.networking.k8s.io? If not, remove "yet" here

```suggestion
apiVersion: networking.gke.io/v1 # Specific to GKE; not available in gateway.networking.k8s.io
```